zz/zzhoracerta.sh

```diff
@@ -77,8 +77,10 @@ zzhoracerta ()
 
 	# Faz a consulta e filtra o resultado
 	zztool dump "$url/current_time_in_$localidade.aspx" |
-		sed -n '/Current Time in /,/Daylight Saving Time:/{
+		sed -n '/Current Time in /,/^UTC/{
 			s/Current Time in //
+			/^\[social\]$/d
+			/^What Time Is It/d
```

Comment:
Localmente funcionou certo, mas no Travis continua dando o mesmo problema de antes:

```
[FAILED #234, line 23] zzhoracerta rio grande do sul | sed 's/[0-9]/N/g; s/^N:/NN:/; s/ [AP]M$//; s/.* N\{1,2\}, NNNN$/DATA/; s/ Summer//; s/BRST/BRT/'
@@ -1,5 +1,718 @@
 Rio Grande do Sul, Brazil
+What Time Is It In Rio Grande do Sul, Brazil?Local Time
 NN:NN:NN
 DATA
 Brasília Time (BRT) -NNNN UTC
 DATA
+* DST
+* Time Zone Map
+Daylight Saving Time
+Rio Grande do Sul, Brazil time zone location map borders
+This location does not observe Daylight Saving Time.
+* Cheap Hotels
+* Cheap Flights
+* Nearby Airports
+* Cheap Hotels
+* Cheap Flights
+* Nearby Airports
+* Cheap Hotels
+* Cheap Flights
+* Nearby Airports
```

Estranho, pelo menos essa linha do "What Time Is It" tinha que ter sido removida.

Reply:
Que tal usar ao invés de ` zztool dump ` usar no lugar ` zztool source `:
```
	zztool source "$url/current_time_in_$localidade.aspx" |
		sed -n '/Current Time in /,/^ *<\/p>/p' |
		zztrim |
		zzxml --untag |
		sed '
			s/Current Time in //
			/^What Time Is It/d
			/^ *$/d
			s/^ *//
		' |
		zzjuntalinhas -i 5 -f 8 -d ' '
```
Talvez possa eliminar o ` zztrim ` e o ` zzxml ` dentro de um sed único.
Essa construção funcionou bem comigo..

Reply:
Agora acho que funcionou:
https://travis-ci.com/github/funcoeszz/funcoeszz/builds/211250537#L283

Reply:
Show @itamarnet, sua versão ficou mais fácil de entender (e funciona 😄 )